subjects/blockchain/financial-instruments/audit/README.md

```diff
@@ -0,0 +1,35 @@
+## Financial Instruments Audit
+
+##### Read the documentation
+###### Does the README file contain or redirect to the instructions to launch the project?
+###### Are the instructions to install and deploy the project usable to you?
+###### Does the documentation contain a description of the architecture of the project?
+###### Does the documentation contain details on the API and smart contract functions?
+
+##### Launch the project
+###### Is a blockchain network running?
+###### Can you display the web interface?
+###### Does the homepage allow you to connect a wallet?
+
+##### Connect a wallet
+###### Were you asked to provide a name and upload a passport picture?
+###### Does your wallet appear to be connected?
+
+##### Use the provided script to populate the platform
+###### Was the documentation clear enough to launch the populating script?
+###### Does the website show the various digital assets?
+###### Did the two accounts receive the corresponding asset looking at the portfolio page once connected?
+
+##### As the first user, Aya, opens the CLV page and offers to sell 10 CLV at 9 TRG
+###### Does the interface reflect the amount of CLV available to Aya? 
+###### Does the order validation trigger a transfer of funds?
+
+##### As the second user, Beatriz, open the CLV page and buys 10 CLV at market price
+###### Does the platform correctly offer to buy at 9 TRG?
+###### Does the trade seem to execute?
+###### Does Beatriz have 60 TRG and 10 CLV now?
+###### Does Beatriz portfolio display a withdrawal button for CLV?
+
+##### From the portfolio page, click on withdraw for CLV
+###### Does the transaction execute?
+###### Does Beatriz now have full possession of its assets (in the second column)?
```

Comment:
Usually there is a line spacing between all the lines in the audits - it seems to not affect the final rendering, but it would be nice to keep the style consistent in my opinion

Reply:
done